$OpenBSD$

revert long-distance matching in zstd which is based on private functions and
doesn't compile if archivers/zstd is installed
this is a revert of pertinent parts of commit 
f3436a841a3b59fc3a9deb94d5fdfd1e550d0351
based on github issue #17374

Index: core/io/compression.h
--- core/io/compression.h.orig
+++ core/io/compression.h
@@ -39,8 +39,6 @@ class Compression { (public)
 	static int zlib_level;
 	static int gzip_level;
 	static int zstd_level;
-	static bool zstd_long_distance_matching;
-	static int zstd_window_log_size;
 
 	enum Mode {
 		MODE_FASTLZ,
